src/commonControls/numberControl/NumberControl.ts

```diff
@@ -594,6 +670,33 @@ export class NumberControl extends Control implements InteractionModelContributo
         }
     }
 
+    private isScreenEvent(input: ControlInput) {
+        try {
+            okIf(InputUtil.isAPLUserEventWithMatchingControlId(input, this.id));
+            this.handleFunc = this.handleScreenEvent;
```

Comment:
best to also test that there are two parameters in total so that the ! on 684 is safe.